modules/gatk/indelrealigner/main.nf

```diff
@@ -0,0 +1,54 @@
+process GATK_INDELREALIGNER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk=3.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk:3.5--hdfd78af_11':
+        'quay.io/biocontainers/gatk:3.5--hdfd78af_11' }"
+
+    input:
+    tuple val(meta), path(input), path(index), path(intervals)
```

Comment:
Reading the docs, ( I only found some html files that are not displayed nicely on github. i guess they don't display them on the website anymore before 4.0.0); I understood that it only takes bam files. In that case I think it would make sense to be explicit here. (Or if I misread then the output would also need to account for cram files)

Reply:
Correct, good point! Will call just BAM/BAI

Yeah they did a bad job of 'deprecating' GATK<= 3 :\